isle-drush_make/drupal.drush.make

```diff
@@ -23,7 +23,8 @@ projects[panels][subdir] = "contrib"
 projects[imagemagick][subdir] = "contrib"
 projects[libraries][subdir] = "contrib"
 projects[link][subdir] = "contrib"
-projects[nodequeue][subdir] = "contrib"
+; 18-Nov-2019 - nodequeue is unstable/unsupported. MAM removing it to fix build/install errors.
+; projects[nodequeue][subdir] = "contrib"
```

Comment:
After talking with folks like you, Noah and Gavin, we don't really need nodequeue and I think these lines can just come out.   We don't need to leave them in here at all, since we have git history if we need to see what's changed.